src/gaTracker/colibrius-g.ts

```diff
@@ -16,17 +10,14 @@ declare let window: CustomWindow;
 
 export const loadGaTracker = async (): Promise<void> => {
 	LittledataPixel.init({ pixelId: window.LittledataLayer.webPropertyId });
-	await getTrackingConsent('google').then(() => {
+	await getTrackingConsent('google').then(async () => {
 		initGtag();
 		advertiseLD('Google Analytics');
-		retrieveAndStoreClientId();
-		retrieveAndStoreGA4Id();
-		pageView(function () {
-			window.ga(function () {
-				window.LittledataLayer.isDawnThemeActive = window?.Shopify?.theme?.name === 'Dawn';
-				sendPageview();
-				trackEvents();
-			});
+		await storeGaIdsToTM();
+		pageView(() => {
+			window.LittledataLayer.isDawnThemeActive = window?.Shopify?.theme?.name === 'Dawn';
+			sendPageview();
+			trackEvents();
 		});
```

Comment:
```suggestion
	await getTrackingConsent('google');
	initGtag();
	advertiseLD('Google Analytics');
	await storeGaIdsToTM();
	pageView(() => {
		window.LittledataLayer.isDawnThemeActive = window?.Shopify?.theme?.name === 'Dawn';
		sendPageview();
		trackEvents();
	});
```

I suppose at this point you might as well do this

Reply:
good point. done